elftools/dwarf/enums.py

```diff
@@ -11,79 +11,82 @@
+    DW_TAG_common_inclusion            = 0x1b,
+    DW_TAG_inheritance                 = 0x1c,
+    DW_TAG_inlined_subroutine          = 0x1d,
+    DW_TAG_module                      = 0x1e,
+    DW_TAG_ptr_to_member_type          = 0x1f,
+    DW_TAG_set_type                    = 0x20,
+    DW_TAG_subrange_type               = 0x21,
+    DW_TAG_with_stmt                   = 0x22,
+    DW_TAG_access_declaration          = 0x23,
+    DW_TAG_base_type                   = 0x24,
+    DW_TAG_catch_block                 = 0x25,
+    DW_TAG_const_type                  = 0x26,
+    DW_TAG_constant                    = 0x27,
+    DW_TAG_enumerator                  = 0x28,
+    DW_TAG_file_type                   = 0x29,
+    DW_TAG_friend                      = 0x2a,
+    DW_TAG_namelist                    = 0x2b,
+    DW_TAG_namelist_item               = 0x2c,
+    DW_TAG_namelist_items              = 0x2c,
+    DW_TAG_packed_type                 = 0x2d,
+    DW_TAG_subprogram                  = 0x2e,
 
     # The DWARF standard defines these as _parameter, not _param, but we
     # maintain compatibility with readelf.
-    DW_TAG_template_type_param      = 0x2f,
-    DW_TAG_template_value_param     = 0x30,
+    DW_TAG_template_type_param         = 0x2f,
+    DW_TAG_template_value_param        = 0x30,
 
-    DW_TAG_thrown_type              = 0x31,
-    DW_TAG_try_block                = 0x32,
-    DW_TAG_variant_part             = 0x33,
-    DW_TAG_variable                 = 0x34,
-    DW_TAG_volatile_type            = 0x35,
-    DW_TAG_dwarf_procedure          = 0x36,
-    DW_TAG_restrict_type            = 0x37,
-    DW_TAG_interface_type           = 0x38,
-    DW_TAG_namespace                = 0x39,
-    DW_TAG_imported_module          = 0x3a,
-    DW_TAG_unspecified_type         = 0x3b,
-    DW_TAG_partial_unit             = 0x3c,
-    DW_TAG_imported_unit            = 0x3d,
-    DW_TAG_mutable_type             = 0x3e,
-    DW_TAG_condition                = 0x3f,
-    DW_TAG_shared_type              = 0x40,
-    DW_TAG_type_unit                = 0x41,
-    DW_TAG_rvalue_reference_type    = 0x42,
+    DW_TAG_thrown_type                 = 0x31,
+    DW_TAG_try_block                   = 0x32,
+    DW_TAG_variant_part                = 0x33,
+    DW_TAG_variable                    = 0x34,
+    DW_TAG_volatile_type               = 0x35,
+    DW_TAG_dwarf_procedure             = 0x36,
+    DW_TAG_restrict_type               = 0x37,
+    DW_TAG_interface_type              = 0x38,
+    DW_TAG_namespace                   = 0x39,
+    DW_TAG_imported_module             = 0x3a,
+    DW_TAG_unspecified_type            = 0x3b,
+    DW_TAG_partial_unit                = 0x3c,
+    DW_TAG_imported_unit               = 0x3d,
+    DW_TAG_mutable_type                = 0x3e,
+    DW_TAG_condition                   = 0x3f,
+    DW_TAG_shared_type                 = 0x40,
+    DW_TAG_type_unit                   = 0x41,
+    DW_TAG_rvalue_reference_type       = 0x42,
```

Comment:
Maybe it's the review tool, but the `=`s don't seem all aligned now. Why did you reformat the ones above if the ones below are on a different offset anyway?

Reply:
Hmm, I think it might be the review tool -- `_default_` is now unaligned (I'll fix that), but the others were aligned before and after my changeset. In any case, I can drop the alignment changes if you'd like.

Reply:
Weird. `_default_` is aligned in my local editor, but not on GitHub's review tool 😕